impl/maven-cli/src/main/java/org/apache/maven/cling/invoker/LookupInvoker.java

```diff
@@ -407,11 +407,7 @@ protected Consumer<String> doDetermineWriter(C context) {
         } else {
             // Given the terminal creation has been offloaded to a different thread,
             // do not pass directly the terminal writer
-            return msg -> {
-                PrintWriter pw = context.terminal.writer();
-                pw.println(msg);
-                pw.flush();
-            };
+            return msg -> context.terminal.writer().println(msg);
```

Comment:
What's the purpose of removing the `flush` call here ?

Reply:
delegation, therefore considered boilerplate. 

it will be flushed, when closed triggered within finallize by `gc`. 

we should not make explicit, what already there (done by someone else).

Reply:
- https://github.com/apache/maven/pull/2456